execution/executor-benchmark/src/transaction_generator.rs

```diff
@@ -188,7 +201,9 @@ impl TransactionGenerator {
 
     pub fn run_mint(&mut self, init_account_balance: u64, block_size: usize) {
         assert!(self.block_sender.is_some());
+        self.gen_nft_initialization();
         self.gen_account_creations(block_size);
+        self.gen_account_registrations(block_size);
```

Comment:
wow, this will take forever if we want a large DB 😂

Reply:
Yeah, but it is not evitable